### PR TITLE
Use v2 context for the data-integrity

### PR DESCRIPTION
**Pull request by iherman**
See title

***
<a href="https://pr-preview.s3.amazonaws.com/w3c/vc-di-bbs/pull/147.html" title="Last updated on Apr 28, 2024, 3:39 PM UTC (c034cf8)">Preview</a> | <a href="https://pr-preview.s3.amazonaws.com/w3c/vc-di-bbs/147/a63511e...c034cf8.html" title="Last updated on Apr 28, 2024, 3:39 PM UTC (c034cf8)">Diff</a>

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -414,7 +414,7 @@ <h3>Multikey</h3>
 {
   "@context": [
     "https://www.w3.org/ns/did/v1",
-    "https://w3id.org/security/data-integrity/v1"
+    "https://w3id.org/security/data-integrity/v2"
```
Comment (dlongley) [inline index.html:417]: This example DID doc should actually be using the multikey context for the multikey terms in use below:

```suggestion
    "https://w3id.org/security/multikey/v1"
```

**Review by dlongley — APPROVED** [review]
Approving if my suggestion is accepted.

**Comment by iherman** [comment]
Note: This shoud not be merged before the 4th of April, ie, before the publication of the CR Snapshot!!!!

**Review by Wind4Greg — APPROVED** [review]

**Comment by msporny** [comment]
Editorial, multiple reviews, changes requested and made, no objections, merging.

**Pull request merged**